torchmd/parameters.py

```diff
@@ -32,8 +32,10 @@ def __init__(
         self.to_(device)
 
     def to_(self, device):
-        self.A = self.A.to(device)
-        self.B = self.B.to(device)
+        if self.A is not None:
+            self.A = self.A.to(device)
+        if self.B is not None:
+            self.B = self.B.to(device)
```

Comment:
Nice! Would you mind grouping them under a single if though? They cannot exist individually so it's clearer if we put them under a single if.
```
if self.A is not None and self.B is not None:
    ...
```
same in `def precision` function.

Reply:
Also for my OCD reasons could you move the self.charges and self.masses to the top so that all the if's are together? :D Same also in `def precision_`.
That's all.

Reply:
Usually in a .prmptop file there are two separate sections: one for the A coeff (repulsive term) and one for the B coeff (attractive term). In our case, when forceterms = [], there are no problems because we don't have both. 

If you want to work with prior maybe you could have the A coeff. and not the B coeff. in the prmtop file (or viceversa) but in this case I don't know if is possible to calculate the LJ so at the you're right: it's OK the single if.

Reply:
ok, leave it as is then. I could not think of a case of individual A/B but it makes sense. Although from what I see they are calculated together
```py
        if "lj" in terms or "LJ" in terms:
            self.A, self.B = self.make_lj(ff, uqatomtypes)
```
But it's more future-proof to have them separate I guess. Fine

Reply:
Move the charges and masses to the top though if you don't mind :D Or I'll do it myself after the PR